configure

```diff
@@ -1909,6 +1909,12 @@ asm_cfi_supported=false
 
 export as aspp
 
+if $as --debug-prefix-map old=new simple.S; then
```

Comment:
I suppose that this will write on stderr in the case where it does not work. Is it the expect behavior, or do other `tryfoo.sh` tests typically silence the failure output?

Reply:
One would get a message akin to:
```
as: unrecognized option '--debug-prefix-map'
```
I don't think it is a major usability issue, but for the sake
of consistency, I added a redirection.